template-parts/acf-media.php

```diff
@@ -21,7 +21,7 @@
 <figure class="c-media">
 
     <div class="c-media__masthead">
-        <img src="<?php echo $thumb; ?>" alt="<?php echo $alt; ?>"  class="c-media__media">
+        <img src="<?php echo $thumb; ?>" alt="<?php echo $alt; ?>"  class="c-media__media" width=100% height=auto>
```

Comment:
Should width and height values be wrapped in ""?

Reply:
According to [this w3schools article](https://www.w3schools.com/html/html_attributes.asp), the HTML5 standard does not require quotes around attribute values but W3C recommends quotes in HTML and demands quotes for stricter document types like XHTML. On that basis I think we should use quotes and I shall update the code.